docs/07-BDD.md

```diff
@@ -1,90 +1,113 @@
 * write actual code implementing the feature
 
-By writing every feature in User Story format that is automatically executable as a test we ensure that: business, developers, QAs and managers are in the same boat.
+By writing every feature in a User Story format that is automatically executable as a test we ensure that:
+business, developers, QAs and managers are on the same page.
 
-BDD encourages exploration and debate in order to formalize the requirements and the features that needs to be implemented by requesting to write the User Stories in a way that everyone can understand.
+BDD encourages exploration and debate in order to formalize the requirements and the features
+that need to be implemented by requiring writing the User Stories in a way that everyone can understand.
 
-By making tests to be a part of User Story, BDD allows non-technical personnel to write (or edit) Acceptance tests.
+By making tests be a part of the User Story, BDD allows non-technical personnel to write (or edit) Acceptance tests.
 
-With this procedure we also ensure that everyone in a team knows what has been developed, what has not, what has been tested and what has not.
+With this procedure we also ensure that everyone in a team knows what has been developed,
+what has not, what has been tested and what has not.
 
 ### Ubiquitous Language
 
-The ubiquitous language is always referred as *common* language. That is it's main benefit. It is not a couple of our business specification's words, and not a couple of developer's technical terms. It is a common words and terms that can be understood by people for whom we are building the software and should be understood by developers. Establishing correct communication between this two groups people is vital for building successful project that will fit the domain and fulfill all business needs.
+The ubiquitous language is always referred as *common* language. That is its main benefit.
+It is not some of our business specification's words, and some of the developer's technical terms.
+It is a common words and terms that can be understood by people for whom we are building the software
+and should be understood by developers. Establishing correct communication between these two groups people is vital
+for building a successful project that will fit the domain and fulfil all business needs.
 
-Each feature of a product should be born from a talk between
+Each feature of a product should be born from a talk between:
 
 * business (analysts, product owner)
 * developers
 * QAs
 
-which are known in BDD as "three amigos".
+who are known in BDD as the "three amigos".
 
-Such talks should produce written stories. There should be an actor that doing some things, the feature that should be fulfilled within the story and the result achieved. 
+Such talks should produce written stories. There should be an actor that want to do some things,
+the feature that should be fulfilled within the story and the result achieved.
 
```

Comment:
Those talks should produce written User Stories. Each User Story should include:

* **an actor** (who makes the action)
* a wished feature (basically the action)
* a result to be achived